CHANGELOG.md

```diff
@@ -4,8 +4,11 @@ OAuth 2.0 Google Provider Changelog
 
 ### Changed
 
-- Add PHP 8.0 support, require PHP 7.3 or newer
-- Add types where possible
+- Add GitHub Actions instead Travis, #96 by @yozhef
+- Update Code Style level to PSR-12, #97 by @yozhef
```

Comment:
These (and Coveralls) aren't functional changes. I'm sorry that I forgot to credit you, I will make that change.